table/sst_file_reader_test.cc

```diff
@@ -37,14 +39,7 @@ class SstFileReaderTest : public testing::Test {
     sst_name_ = test::PerThreadDBPath("sst_file");
 
     Env* base_env = Env::Default();
-    const char* test_env_uri = getenv("TEST_ENV_URI");
```

Comment:
I think we should use CreateFromFlags here and specify the input. Instead of give CreateFromSystem. We may not need CreateFromSystem as the API.